tg_bot/modules/warns.py

```diff
@@ -79,7 +79,7 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
                                                   mention_html(user.id, user.first_name), 
```

Comment:
if you move the call to format, it's your job to align the OTHER arguments too. Not just the ones you've added.

Reply:
consider the consistency.